examples/healthcare/brain_anomaly_detection/train_FNO.py

```diff
@@ -0,0 +1,153 @@
+import modulus
+import modulus.sym
+from modulus.sym.hydra import to_absolute_path
+from modulus.sym.distributed.manager import DistributedManager
+
+import torch
+import numpy as np
+import os
+from torch.utils.data import DataLoader, Dataset
+from typing import Union
+import h5py
+import numpy as np
+import hydra
+from omegaconf import DictConfig
+from modulus.models.fno import FNO
+from torch.utils.data import Dataset, DataLoader
+from modulus.launch.logging import PythonLogger, LaunchLogger
+from torch.nn import MSELoss
+from torch.optim import Adam, lr_scheduler
+from modulus.launch.utils import load_checkpoint, save_checkpoint
+import torch.nn.functional as F
+
+class HDF5MapStyleDataset(Dataset):
+    """Simple map-stype HDF5 dataset"""
+
+    def __init__(
+        self,
+        file_path,
+        device: Union[str, torch.device] = "cuda",
+    ):
+        self.file_path = file_path
+        with h5py.File(file_path, "r") as f:
+            self.keys = list(f.keys())
+        # Set up device, needed for pipeline
+        if isinstance(device, str):
+            device = torch.device(device)
+        # Need a index id if cuda
+        if device.type == "cuda" and device.index == None:
+            device = torch.device("cuda:0")
+        self.device = device
+    def __len__(self):
+        with h5py.File(self.file_path, "r") as f:
+            return len(f[self.keys[0]])
+    def __getitem__(self, idx):
+        data = {}
+        with h5py.File(self.file_path, "r") as f:
+            for key in self.keys:
+                data[key] = np.array(f[key][idx])
+        invar = torch.from_numpy(data["wavefield_in"])
+        outvar = torch.from_numpy(data["wavefield_sol"])
+        if self.device.type == "cuda":
+            # Move tensors to GPU
+            invar = invar.cuda()
+            outvar = outvar.cuda()
+
+        return invar, outvar
+
+
+@hydra.main(version_base="1.3", config_path="./conf/", config_name="config_FNO_launch")
+def main(cfg: DictConfig) -> None:
+    DistributedManager.initialize()  # Only call this once in the entire script!
+    dist = DistributedManager()  # call if required elsewhere
+
+    # initialize monitoring
+    logger = PythonLogger(name="brain_fno")
+
+    # initialize monitoring
+    LaunchLogger.initialize()
+
+    ### get the data
+    train_path = to_absolute_path(
+        "./train_sets/data_scale_train.hdf5"
+    )
+    train_dataset = HDF5MapStyleDataset(train_path, device="cuda")
+    train_dataloader = DataLoader(
+        train_dataset, batch_size=cfg.training.batch_size, shuffle=True
+    )
+
+    test_path = to_absolute_path(
+        "./train_sets/data_scale_test.hdf5")
+    # rewrite this into class
+    test_dataset = HDF5MapStyleDataset(test_path, device="cuda")
+    test_dataloader = DataLoader(
+    test_dataset, batch_size=cfg.test.batch_size, shuffle=False
+    )
+    # set device as GPU
+    device = "cuda"
```

Comment:
same comment here. Since you are using the distributed manager from Modulus, you can just do `dist.device`